query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/alreadyConverted/NestedDeleteMutationInsideUpsertSpec.scala

```diff
@@ -7,63 +7,6 @@ import util._
 class NestedDeleteMutationInsideUpsertSpec extends FlatSpec with Matchers with ApiSpecBase with SchemaBaseV11 {
   override def runOnlyForCapabilities = Set(JoinRelationLinksCapability)
 
-  "a P1! to C1! relation " should "error when deleting the child" in {
```

Comment:
This looks like a dedicated test case for required 1:1 relations that we don't need anymore.

Reply:
Jep, that can go.